### PR TITLE
Maurice/fix

### PR DESCRIPTION
**Pull request by Barnes2197**
deployed a route to update all emails to lower case. has admin auth to prevent unintended outside use

### CODE REVIEW

**Comment by Tommy2016x** [comment]
Does the email lowercase really need to be an actual route? why not just run a script in a local instance or something once

**Comment by mrmcpowned** [comment]
I agree with @Tommy2016x, we shouldn't need a route to do this. It should be run as a script. We do need to make sure the API lowercases any incoming emails though, else there's going to be issues when accessing and modifying information based on an email address.

**Comment by Barnes2197** [comment]
With the way things are being built with backpack idk how I would be able to run it as a script. Without automatically calling it on startup. So it was either make it a route or include it as an import into index.js as a one time run on start.

**Comment by Barnes2197** [comment]
So I was able to do it but literally starting the db in the script updating the emails then closing the connection. Is that how it should be run???

File: src/lowerCaseEmails.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+  useFindAndModify: false
+};
+
+const db = () => Promise.resolve(mongoose.connect(MONGO_URI, options));
+
+db()
+  .then(() => console.log("> 🍃  Mongo connected"))
+  .catch(e => console.log("> Mongo error ", e.message));
+
+  const applicantSchema = new mongoose.Schema({
```
Comment (Tommy2016x) [inline src/lowerCaseEmails.js]: you shouldn't be making a new mongoose schema here, just import the already created one in the models folder

File: src/lowerCaseEmails.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+const {MONGO_URI} = process.env;
+
+const options = {
+  useNewUrlParser: true,
+  useCreateIndex: true,
+  dbName: "shellhacks",
+  useFindAndModify: false
+};
+
+const db = () => Promise.resolve(mongoose.connect(MONGO_URI, options));
```
Comment (Tommy2016x) [inline src/lowerCaseEmails.js]: an instance of the db function is already being called in another file, no need to re run it again

File: src/lowerCaseEmails.js [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+    mlh: { type: Boolean, default: false },
+    mlhAffiliation: { type: Boolean, default: false },
+    sponsorPromo: { type: Boolean, default: false },
+    timeCreated: { type: Date },
+    timeApplied: { type: Date }
+});
+
+const Applicant = mongoose.model("Applicant", applicantSchema);
+
+
```
Comment (Tommy2016x) [inline src/lowerCaseEmails.js]: Make sure to import this file in the index js of the /src folder after the 'db' import. Test it out locally with the changes I requested to make sure everything works

**Review by Tommy2016x — CHANGES_REQUESTED** [review]

File: src/lowerCaseEmails.js (hunk 1 above)
Comment (Barnes2197) [reply src/lowerCaseEmails.js]: I tried that and got a syntax error on import.

**Review by Tommy2016x — APPROVED** [review]

**Pull request merged**